web/templates/draft_table_body.hbs

```diff
@@ -6,10 +6,23 @@
                 <div class="exit">
                     <span class="exit-sign">&times;</span>
                 </div>
-                <div class="removed-drafts">
-                    {{t "Drafts are not synced to other devices and browsers." }}
-                    <br />
-                    {{#tr}}Drafts older than <strong>{draft_lifetime}</strong> days are automatically removed.{{/tr}}
+                <div class="header-body">
+                    <div class="removed-drafts">
+                        {{t "Drafts are not synced to other devices and browsers." }}
+                        <br />
+                        {{#tr}}Drafts older than <strong>{draft_lifetime}</strong> days are automatically removed.{{/tr}}
+                    </div>
+                    <div class="delete-drafts-group">
+                        <div class="delete-selected-drafts-button-container message_control_button">
```

Comment:
What's the thinking behind using the `message_control_button` class here? I worry that the reuse of this class in just this one place is more likely to result in bugs where we accidentally break this when adjusting message feed design.

Reply:
I thought of using this class to make sure the tippy on the button is removed after the keyboard focus is gone to some other button. But as you mentioned using this class here can lead to bugs. 

I will do the following change in [tippyjs.js](https://github.com/zulip/zulip/blob/main/web/src/tippyjs.js#L146) and remove the above mentioned class:
```diff
-    $("body").on("blur", ".message_control_button", (e) => {
-        // Remove tooltip when user is trying to tab through all the icons.
-        // If user tabs slowly, tooltips are displayed otherwise they are
-        // destroyed before they can be displayed.
-        e.currentTarget?._tippy?.destroy();
-    });
+    $("body").on(
+        "blur",
+        ".message_control_button, .delete-selected-drafts-button-container",
+        (e) => {
+            // Remove tooltip when user is trying to tab through all the icons.
+            // If user tabs slowly, tooltips are displayed otherwise they are
+            // destroyed before they can be displayed.
+            e.currentTarget?._tippy?.destroy();
+        },
+    );

```

Reply:
Yeah that sounds great.